install_files/ansible-base/inventory-dynamic

```diff
@@ -37,16 +47,17 @@ def lookup_local_ipv4_address(hostname):
             monitor_ip = site_vars['monitor_ip']
 
     except (IOError, KeyError):
-        msg = "You must run 'secure-admin sdconfig' from root securedrop dir."
-        raise Exception(msg)
+        msg = ("Run `./securedrop-admin sdconfig` to configure"
```

Comment:
Hmm you're right, the new message is a bit _less_ clear in terms of the root of the repo... but the old text was `secure-admin sdconfig` which was the wrong script name, and that's mostly what I wanted to tackle here.